Insights/InputEntity_Host/ActionsOnAccounts.yaml

```diff
@@ -103,4 +110,22 @@ Insights:
  
  AdditionalQuery:
   Text: "See All Actions on Accounts"
-  Query: "project TimeGenerated, EventID, Activity, Computer, TargetAccount, TargetUserName, TargetDomainName, TargetSid, SubjectUserName, SubjectUserSid | order by TimeGenerated desc"
+  Query: "project TimeGenerated, EventID, Activity, Computer, TargetAccount, TargetUserName, TargetDomainName, TargetSid, SubjectUserName, SubjectUserSid, _ResourceId, SourceComputerId, timestamp, HostCustomEntity, AccountCustomEntity | order by TimeGenerated desc"
+
+Activities:
+ enabledByDefault: true
+ items: 
+   - id: 307c85ee-39a2-4da3-952e-4fd79aa46d3a
+     description: Account created on Host
+     title: "An account was created on this host"
+     content: "On '{{Computer}}' the account '{{TargetAccount}}' was created by '{{AddedBy}}'"
```

Comment:
This is a matter of style, but since this is {{Computer}}'s entity page do we need to output the computer?

Reply:
I think we have both ways and let customers provide us feedback, we have some private preview customers who will do just that. :)